etcdctl/ctlv3/command/ep_command.go

```diff
@@ -88,7 +89,7 @@ type epHealth struct {
 
 // epHealthCommandFunc executes the "endpoint-health" command.
 func epHealthCommandFunc(cmd *cobra.Command, args []string) {
-	lg, err := zap.NewProduction()
+	lg, err := logutil.CreateDefaultZapLogger(zap.InfoLevel)
```

Comment:
why not use `etcdClientDebugLevel()` here? I see other places all use it.

Reply:
Got it, it's in different package `client/v3`

Reply:
etcdClientDebugLevel is only used by etcd client in `client/v3` package. The default verbosity for `zap.NewProduction` and `zap.NewExample` is `info`. I didn't want this PR to make any changes as this is a fix that I plan to backport to v3.5